Basic-Car-Maintenance/Documentation.docc/Documentation.md

```diff
@@ -16,3 +16,76 @@ Icons have the following colors:
 * Blue #253AA7
 * Black #000000
 * Orange #F78616
+
+## Firestore Collections
+
+**_alerts_**  
+The alerts collection contains system level alerts that will be visable to all users
+
+| Fields    | Type    |
+| --------- | ------- |
+| actionTex | Text    |
+| actionURL | Text    |
+| emojiIcon | Text    |
+| id        | Text    |
+| isOn      | Boolean |
+| message   | Text    |
+| title     | Text    |
+
+**_maintainance_events_**  
+Maintainance events collection contains the individual maintainance events that are associated
+with a specific user.
+
+| Fields    | Type       |
+| --------- | ---------- |
+| id        | DocumentID |
+| userID    | Text       |
+| title     | Text       |
+| date      | Date       |
+| notes     | Text       |
+| vehicleID | Text       |
+
+**_vehicles_**  
+The vehicles collection contains all the vehicles associated with a specific user.
+
+| Fields             | Type       |
+| ------------------ | ---------- |
+| id                 | DocumentID |
+| userID             | Text       |
+| name               | Text       |
+| make               | Text       |
+| model              | Text       |
+| year               | Text       |
+| color              | Text       |
+| vin                | Text       |
+| licensePlateNumber | Text       |
+
+## Firestore Rules
+
+**alerts** : read-only for all users
+
+**maintainance_events** : Authorized users can read and write to the
+maintainance events collection that is associated with their user id.
+
+**vehicles**: Authorized users can ready and write to vehicles collection that is associated with their user id.
+
+```
+rules_version = '1';
+service cloud.firestore {
+match /databases/{database}/documents {
+
+        match /alerts/{document=\*\*} {
+        allow read;
+        }
+
+        match /maintainance+events/{allPaths=**} {
+        allow read, write: if request.auth != null;
+        }
```

Comment:
This allows anyone to read anyone's data, right?
you should only be allowed to read your own data

Reply:
this hasn't changed, is this the correct syntax? Can you link the Firestore documentation for this? @rramirez-dev